packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -855,10 +864,22 @@ public String stopVideoRecording() {
     try {
       closeRenderer();
       captureSession.abortCaptures();
+    } catch (CameraAccessException e) {
+      // Ignore exceptions and try to continue (changes are camera session already aborted capture).
+    }
+
+    boolean haveCaptureError = false;
+
+    try {
       mediaRecorder.stop();
-    } catch (CameraAccessException | IllegalStateException e) {
+    } catch (IllegalStateException e) {
       // Ignore exceptions and try to continue (changes are camera session already aborted capture).
+    } catch (RuntimeException e) {
+      // We marked that mediaRecorder.stop() is failed when no valid audio/video data has been
```

Comment:
Please don't use "we" in comments; it's not clear to me who "we" is here.

What is this comment referring to exactly? What does "marked [...] is failed" mean, and where does that happen?